src/backend/catalog/pg_proc.c

```diff
@@ -125,6 +125,11 @@ ProcedureCreate(const char *procedureName,
 	 */
 	Assert(PointerIsValid(prosrc));
 
+	if (pltsql_get_object_owner_hook)
```

Comment:
Let's add `!OidIsValid(proowner)` here as well.

Reply:
proowner is guaranteed to be valid here.
The reason why I used it in DefineRelation is to skip our hook for case of sequences created for identity columns. Those should be owned by whoever was the table owner.